planet/cli/subscriptions.py

```diff
@@ -19,20 +20,66 @@
 _fake_sub_results: Dict[str, list] = {}
 
 
-# The following 5 functions are for use in CLI commands until there
-# is subscriptions Python client to call on.
-def _count_fake_subs():
-    return len(_fake_subs)
+class PlaceholderSubscriptionsClient:
+    """A placeholder client.
 
+    This class and its methods are derived from tests of a skeleton
+    Subscriptions CLI.
```

Comment:
Important note: the only thing that's roughly right about this placeholder client is the method names and signatures. All of the code inside the methods will be very different in the future production client.